docker/.env.aggron

```
@@ -48,3 +48,17 @@ REACT_APP_NERVOS_EXPLORER_URL=https://explorer.nervos.org/aggron
 REACT_APP_NETWORK=Aggron
 REACT_APP_CKB_NODE_URL=https://testnet.ckb.dev/rpc
 REACT_APP_CKB_INDEXER_URL=https://testnet.ckb.dev/indexer
+
+## exchange configs
+# initiated amount of cells for exchange
+CELL_AMOUNT=100
```

```suggestion
EXCHANGE_CELL_COUNT=100
```

the `amount` here is confused, maybe use the `count` here is more appropriate